apps/storybook-react/package.json

```diff
@@ -1,14 +1,15 @@
 {
   "name": "@metamask/storybook-react",
-  "version": "0.0.0",
   "private": true,
   "description": "storybook react",
+  "files": [],
```

Comment:
Part of the yarn constraints that we do want private non publishable packages to adhere to. See this comment in `yarn.config.cjs` https://github.com/MetaMask/metamask-design-system/blob/main/yarn.config.cjs#L172-L176